### ISSUE TITLE
Synchronize encoder motors

### ISSUE THREAD

**Issue by fecavy** [harbaum/ftduino#5]
Hi,

I would like to synchronize two fischertechnik encoder motors. Are there any plans for adding this feature to the Ftduino library?

Thank you

**Comment by harbaum** [comment]
Yes, the plans are there. But I don't know when this will be added.
